google/cloud/forseti/notifier/notifiers/cscc_notifier.py

```diff
@@ -257,8 +255,6 @@ def _send_findings_to_cscc(self, violations, source_id=None):
                     client.update_finding(finding,
                                           finding_id,
                                           source_id=source_id)
-                    LOGGER.debug('Successfully updated finding in CSCC:\n%s',
-                                 finding)
```

Comment:
If you are going to remove this here, you should also update the `update_finding()` to make sure that the `LOGGER.debug`  there will capture the same findings details.  i.e. we don't lose any details with this change.

Reply:
These log statement had to be removed as it says created/updated finding even if there is an exception.

Reply:
What I'm suggesting is to just add the `finding` detail to the debug message here:

https://github.com/forseti-security/forseti-security/blob/a9a820feb400dca968eef941fc4d0fc1d8f97ba7/google/cloud/forseti/common/gcp_api/securitycenter.py#L183